infrastructure/web3-adapter/src/db/mapping.db.ts

```
@@ -0,0 +1,201 @@
+import sqlite3 from "sqlite3";
+import { join } from "path";
+import { promisify } from "util";
+
+export class MappingDatabase {
+    private db: sqlite3.Database;
+    private runAsync: (sql: string, params?: any) => Promise<void>;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)